src/main/scala/io/iohk/ethereum/consensus/atomixraft/AtomixRaftConfig.scala

```diff
@@ -45,8 +45,10 @@ object AtomixRaftConfig extends Logger {
       case _ ⇒ throw new IllegalArgumentException("parts.length != 1, 2, 3")
     }
 
+  //noinspection ScalaStyle
   def parseNode(id_host_port: String): AtomixNode = {
     val parts = id_host_port.split(":")
+    log.info(s"atomix-raft: id_host_port = ${id_host_port} => parts = ${parts.toList}")
```

Comment:
Minor: not sure if it's the best place for such logging, as a reader might expect this to be a pure function and want to use it in different context.